datenguidepy/output_transformer.py

```diff
@@ -127,7 +127,7 @@ def _rename_statistic_fields(
 
     @staticmethod
     def _create_statistic_frame(statistic_sub_json: Dict[str, Any]) -> pd.DataFrame:
-        return pd.io.json.json_normalize(statistic_sub_json, sep="_", max_level=1)
+        return pd.json_normalize(statistic_sub_json, sep="_", max_level=1)
```

Comment:
@enryH 
I think this might be interest to you, some stuff was failing (probably after I updated pandas requirements)
This method was moved in pandas so this is a minor fix for that.